tracing-subscriber/benches/fmt.rs

```diff
@@ -57,7 +57,7 @@ fn bench_new_span(c: &mut Criterion) {
                         })
                         .thread(move || {
                             for n in 0..i {
-                                let _span = tracing::info_span!("span", i);
+                                let _span = tracing::info_span!("span", n);
```

Comment:
unrelated change?

Reply:
I mentioned that I'd fix this as well, but I can remove it.

Reply:
that's fine, i forgot.